src/main/g8/$name$.service

```diff
@@ -1,14 +0,0 @@
-[Unit]
-Description=$name;format="cap"$
-
-[Service]
-User=$stack_name_prefix$
-Group=$stack_name_prefix$
-Restart=no
```

Comment:
I don't feel fervently about it one way or the other to be honest. You'll notice if it can't start and if it can restart in the event of a JVM crash without it being noticeable, then I don't think that's disastrous.